catalogue_pipeline/reindex_job_creator/src/reindex_job_creator.py

```diff
@@ -0,0 +1,36 @@
+# -*- encoding: utf-8 -*-
+
+import os
+
+import boto3
+
+from wellcome_aws_utils.dynamo_event import DynamoEvent
+from wellcome_aws_utils.sns_utils import publish_sns_message
+
+
+def main(event, _ctxt):
+    print(f'event={event!r}')
+
+    sns_client = boto3.client('sns')
+    topic_arn = os.environ['TOPIC_ARN']
+
+    for record in event['Records']:
```

Comment:
Might be nice to push this into the utils so we get a generator for dynamo events (like with SNS).

Reply:
It already is, but I couldn't get it to work!